__init__.py

```diff
@@ -15,7 +15,44 @@ class MarkExperience(Event):
 
 
 MODIFIER_REGEX = "[+-]?[0,1,2,3]"
-STAT_REGEX = f"(?:(?:!cool|!tough|!sharp|!charm|!weird) {MODIFIER_REGEX})"
+GAME_STATS = {"motw": ["cool", "tough", "sharp", "charm", "weird"],
+              "pbtastartrek": ["aggressive", "bold", "talk", "tech"]}
+stat_regexes = {}
+
+@match_regex(r"\!set game ?(?P<gamename>.*)", case_sensitive=False)
+@memory_in_event_room
+async def set_game(opsdroid, config, message):
+    game = message.regex.capturesdict()['gamename'][0]
+    if game not in GAME_STATS.keys():
+        message.respond(f"I don't know how to play that. Available options are: {', '.join(GAME_STATS.keys())}")
+        return
+    await opsdroid.memory.put(f"{game}_stats", GAME_STATS[game])
```

Comment:
Also I assume the idea behind storing the actual stats in memory is to allow other sources of configuration for the game definitions in the future?

Reply:
I thought you'd suggested doing this, but possibly I've misremembered that.

Reply:
I think it's a good idea ;) just pondering aloud really.